Dockerfile

```
@@ -0,0 +1,42 @@
+ARG ARCH=amd64
+ARG DISTRO=debian
+ARG DISTRO_VERSION=buster
```

Secondary change needed as a result of changed Docker Hub tags mentioned in the below conversation:

```suggestion
ARG DISTRO_VERSION=10
```